### DISCUSSION TITLE
How to change photos that show up in Photos live tile?

### DISCUSSION THREAD

**Discussion by rmz4ed** [valinet/ExplorerPatcher#1856] [Q&A] [answered]
Hi, I am trying to change the photos that show up in the live tile in the start menu to automatically show the latest ones, as it used to in Windows 10 (See [link](https://answers.microsoft.com/en-us/windows/forum/all/controlling-which-photos-appear-on-the-photo-tile/555a0c7b-6f50-4b4b-89ad-9f06abaea6aa)). Is there a way to do this? If not, is there a way to select which photos show up manually?
Thanks!

**Comment by pyrates999** [comment]
I think this is what you're looking for: https://apps.microsoft.com/store/detail/live-tiles-anywhere/9NR7QQK712PL

**Reply by rmz4ed** [reply]
Thank you, but this seems to only be for the Windows 11 start menu. I am using ExplorerPatcher with the Windows 10 start menu. It shows photos in the live tile, but the photos it shows are old. I am wondering if there is a way to select which photos show in the live tile for the Windows 10 menu in ExplorerPatcher.

**Reply by pyrates999** [reply]
that might be a windows 10 only feature then.

**Reply by rmz4ed** [reply]
It seems to be getting and cycling through photos from some folder, but I am unsure where that folder is located:
![image](https://github.com/valinet/ExplorerPatcher/assets/74930865/ed555829-1bbb-4ff6-89ac-32602ddc200d)

**Reply by rmz4ed** [reply, marked as answer]
Found the folder: C:\Users\[Name]\AppData\Local\Packages\Microsoft.Windows.Photos_8wekyb3d8bbwe\LocalState\PhotosAppTile
Lets you change which photos show up in the live tile. Note that the photos MUST be in .JPG format, or they will show up as grey boxes in the live tile. Not automatic, but still works fine.
